digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/lane/LaneService.scala

```diff
@@ -925,7 +925,7 @@ trait LaneOperations {
   }
 
   def createMultiLanesOnLink(updateNewLanes: Seq[NewLane], linkIds: Set[Long], sideCode: Int, username: String): Seq[Long] = {
-    if (updateNewLanes.size == 0) {
+    if (updateNewLanes.isEmpty()) {
```

Comment:
se tais olla vain updateNewLanes.isEmpty , taisin muistaa syntaksin hieman väärin

Reply:
Ok. Näin taitaa olla.